css/ContactDetailsAvatar.scss

```diff
@@ -57,18 +57,42 @@
 		background-size: cover;
 		// White background for avatars with transparency, also in dark theme
 		background-color: #fff;
+		&:hover + .overlay {
+			opacity: .25;
+		}
 	}
 	&__options {
 		position: absolute;
 		z-index: 2;
 		top: 0;
 		background-color: rgba(0, 0, 0, .2);
 	}
+	.overlay {
+		position: absolute;
+		width: 35px;
+		height: 35px;
+		z-index: 11;
+		cursor: pointer;
+		opacity: 0;
+		border-radius: 50%;
+		background: rgba(0, 0, 0, .5);
+		margin: calc((100% + 15px) / 2);
```

Comment:
Usually not fond of half percentage as this can lead to blurry popups :)

Reply:
I took the code as used in the popovermenu (see line 104). Feel free to adapt.